net/grpcurl/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                grpcurl
+version             1.3.0
+categories          net
+platforms           darwin
+supported_archs     x86_64
+license             MIT
+
+maintainers         @gearnode
+
+checksums           rmd160 479a66bb0634e2d15b92321fcf14454224403128 \
+                    sha256 b800d949ed1184b276bf1d5471211dd60a204e5199db0725efb59a4a09166221
+
+homepage            https://github.com/fullstorydev/grpcurl
+
+description         Like cURL, but for gRPC: Command-line tool for interacting with gRPC servers
+long_description    The main purpose for this tool is to invoke RPC methods on a gRPC server from \
+                    the command-line. gRPC servers use a binary encoding on the wire (protocol buffers, \
+                    or "protobufs" for short). So they are basically impossible to interact with using \
+                    regular curl (and older versions of curl that do not support HTTP/2 are of course \
+                    non-starters). This program accepts messages using JSON encoding, which is much more \
+                    friendly for both humans and scripts.
+
+master_sites        https://github.com/fullstorydev/grpcurl/releases/download/v${version}/
+distname            grpcurl_${version}_osx_${supported_archs}
+dist_subdir         ${name}/${version}
+
+use_tar             yes
+extract.suffix      .tar.gz
+
+use_configure       no
+
+build {
+}
+
+destroot {
+    xinstall ${distpath}/${distfiles} ${destroot}${prefix}/bin/grpcurl
```

Comment:
Is this a shell script? If so, then you can add the following line just under the `license` line. 

`supported_archs           noarch
`

Reply:
@mf2k It's not a shell script, it's a binary/